web/src/js/__tests__/components/Header/MenuToggleSpec.tsx

```diff
@@ -5,19 +5,15 @@ import {
     MenuToggle,
     OptionsToggle,
 } from "../../../components/Header/MenuToggle";
-import { Provider } from "react-redux";
-import { TStore } from "../../ducks/tutils";
-import * as optionsEditorActions from "../../../ducks/ui/optionsEditor";
 import { fireEvent, render, screen, waitFor } from "../../test-utils";
-import { fetchApi, runCommand } from "../../../utils";
```

Comment:
Getting rid of unused imports

![so_good](https://github.com/mitmproxy/mitmproxy/assets/679144/dc3f0c1b-5d2e-42bc-b879-aaed41771e5b)

Reply:
my bad 🙈